webpack.common.js

```diff
@@ -44,17 +44,20 @@ module.exports = {
       ],
       dependOn: 'adhocracy4'
     },
-    // these do not rely on adhocracy and adding the depend causes console error
-    // error possibly due to needing to be loaded at specific time
     blueprint_picker: {
       import: [
         './meinberlin/assets/js/blueprint-picker.js'
       ],
       dependOn: 'adhocracy4'
     },
+    // these do not rely on adhocracy and adding the depend causes console error
+    // error possibly due to needing to be loaded at specific time
     documents: {
       import: './meinberlin/apps/documents/assets/react_documents_init.js'
     },
+    budget_proposals: {
```

Comment:
damn, this is so cool! just realize what you are doing here @phillimorland !
really nice